skypalwarmup/JSUserRegistrationService/src/test/java/application/TestData.java

```diff
@@ -0,0 +1,33 @@
+package application;
+
+/**
+ * Copyright (c) 2016-2018, Jumia.
+ */
+public final class TestData {
+
+    public static final class User {
```

Comment:
Nice way of organization for accessing constants but we can avoid nested classes and simply add all constants directly under TestData class and access them directly like `TestData.CONSTANT_NAME` WDYT?

Reply:
Done